### PR TITLE
README add to run hack/get-terraform.sh

### PR DESCRIPTION
**Pull request by sallyom**
@wking this should prevent a few questions in slack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,11 @@ This will create `bin/openshift-install`. This binary can then be invoked to cre
 bin/openshift-install cluster
 ```
 
+The installer requires the terraform binary alongside openshift-install.
+If you don't have [terraform](https://www.terraform.io/) in `bin/terraform`, run the following:
```
Comment (wking) [inline README.md]: The installer will also look in your `$PATH` ([and your current working directory][1], if you're crazy ;).  And folks are unlikely to have a `bin/terraform` unless they already read this, so maybe focus on the `PATH` and then describe them all afterwards?

```markdown
If you don't have [Terraform](https://www.terraform.io/) in your `$PATH`, run:

``sh
hack/get-terraform.sh
``

to download a copy to `bin/terraform`.  `openshift-install` will invoke `terraform` from:

* The directory in which `openshift-install` lives (so `bin/` if you built `bin/openshift-install` with `hack/build.sh`).
* Your current working directory.
* Your `$PATH`.
```

But maybe that's getting long enough to call for a separate `### Terraform` section?  I dunno.

I'm fine just rolling with what you have if you want.

[1]: https://github.com/openshift/installer/blob/6152bd6735f022c80b3a0f7c68000aca7ab23ee1/pkg/terraform/executor.go#L70-L75

**Comment by sallyom** [comment]
@wking I think I addressed your comment, while keeping the note small enough as to not require a separate section in the README :)

**Comment by wking** [comment]
looks good.  I'll wait until #361 lands and then `/lgtm` this

**Pull request merged**